pegasus/src/social_metadata.rb

```diff
@@ -0,0 +1,155 @@
+      }
+    },
+    "csedweek.org" => {
+      "soon-hoc" => {
+        title: I18n.t(:csedweek_og_title),
+        description: I18n.t(:csedweek_og_description_soon),
+        image: {path: "/images/cs-is-everything-thumbnail.png", width: 1200, height: 627},
+        video: {youtube_key: "QvyTEx1wyOY", width: 640, height: 360}
+      },
+      "week-before-hoc" => {
+        title: I18n.t(:csedweek_og_title),
+        description: I18n.t(:csedweek_og_description_soon),
+        image: {path: "/images/cs-is-everything-thumbnail.png", width: 1200, height: 627},
+        video: {youtube_key: "QvyTEx1wyOY", width: 640, height: 360}
+      },
+      "actual-hoc" => {
+        title: I18n.t(:og_title_here),
+        description: I18n.t(:csedweek_og_description_here),
+        image: {path: "/images/cs-is-everything-thumbnail.png", width: 1200, height: 627},
+        video: {youtube_key: "QvyTEx1wyOY", width: 640, height: 360}
+      },
+      "default" => {
+        title: I18n.t(:csedweek_og_title),
+        description: I18n.t(:csedweek_og_description),
+        image: {path: "/images/cs-is-everything-thumbnail.png", width: 1200, height: 627},
+        video: {youtube_key: "QvyTEx1wyOY", width: 640, height: 360}
+      }
+    },
+    "hourofcode.com" => {
+      "soon-hoc" => {
+        title: I18n.t(:og_title_soon),
+        description: hoc_s(:meta_tag_og_description),
+        description_twitter: hoc_s(:meta_tag_twitter_description),
+        image: {path: "/images/social-media/hourofcode-2015-video-thumbnail.jpg", width: 1440, height: 900},
+        video: {youtube_key: "KsOIlDT145A", width: 640, height: 360}
+      },
+      "week-before-hoc" => {
+        title: I18n.t(:og_title_soon),
+        description: hoc_s(:meta_tag_og_description),
+        description_twitter: hoc_s(:meta_tag_twitter_description),
+        image: {path: "/images/social-media/hourofcode-2015-video-thumbnail.jpg", width: 1440, height: 900},
+        video: {youtube_key: "KsOIlDT145A", width: 640, height: 360}
+      },
+      "actual-hoc" => {
+        title: I18n.t(:og_title_here),
+        description: hoc_s(:meta_tag_og_description),
+        description_twitter: hoc_s(:meta_tag_twitter_description),
+        image: {path: "/images/social-media/hourofcode-2015-video-thumbnail.jpg", width: 1440, height: 900},
+        video: {youtube_key: "KsOIlDT145A", width: 640, height: 360}
+      },
+      "default" => {
+        title: hoc_s(:meta_tag_og_title),
+        description: hoc_s(:meta_tag_og_description),
+        description_twitter: hoc_s(:meta_tag_twitter_description),
+        image: {path: "/images/social-media/hourofcode-2015-video-thumbnail.jpg", width: 1440, height: 900},
+        video: {youtube_key: "KsOIlDT145A", width: 640, height: 360}
+      }
+    },
+    "challenge" => {
+      "soon-hoc" => {
+        title: "Celebrity Challenge",
+        description: "Win a celebrity video chat for your class!",
```

Comment:
i18n for these strings?

Reply:
These were already hardcoded in en, for an en-only page, so this just maintains status quo.